i18n/index.js

```diff
@@ -1,7 +1,39 @@
-require("fs")
-  .readdirSync(__dirname)
-  .filter((file) => file.endsWith(".json"))
-  .forEach((filename) => {
-    const moduleName = filename.split(".")[0];
-    exports[moduleName] = require(`${__dirname}/${filename}`);
-  });
+export default {
```

Comment:
Seeing as I'm importing this file under the `renderer` context where the `fs` module isn't available, https://github.com/replugged-org/replugged/blob/53fba7e2e2c9d6e83e361cc875361d8f842271df/src/renderer/modules/i18n.ts#L4 I had to change this to hardcoded exports otherwise the project won't build.

The only other alternative that I see would be to fetch the i18n files via an IPC handler, I'm not sure if that's considered a better approach.